gatsby-node.js

```diff
@@ -4,78 +4,102 @@
  * See: https://www.gatsbyjs.org/docs/node-apis/
  */
 
-exports.createPages = async function({ actions, graphql }) {
-  
-  /* Generate Learning Center pages */
-  const { data } = await graphql(`
-    query {
-      contentfulLearningCenterSearchPage {
-        categoryButtons {
+/* Generate Learning Center pages */
+const generateLearningPages = async function({ actions, graphql }, locale) {
+  const query = `query {
+    contentfulLearningCenterSearchPage(node_locale: { eq:"` + (locale || 'en-US') + `" } ){
```

Comment:
I discovered that when we're not inside a <StaticQuery> component, we can very easily incorporate dynamic variables into a graphql query. Hooray!

Reply:
Oh cool!  So does that mean that we can close [this clubhouse story](https://app.clubhouse.io/justfixnyc/story/979/investigate-how-to-introduce-variables-into-the-contentful-graphql-queries)?

Reply:
For sure! Although see my note there— the real issue now is that we load in contentful data for each page via the StaticQuery component which strictly forbids any dynamism in the graphql query. Might just need to refactor things :/